app/Api/Puzzles.ts

```diff
@@ -48,6 +49,10 @@ export class Puzzles {
    */
   public static async finishGame(
```

Comment:
The comment about this function says it returns a promise but return type is not typed in function declaration and their is no return statement in the function?

Reply:
Good catch, I fixed the comment.